controllers/dnschaos/types.go

```diff
@@ -117,7 +117,10 @@ func (r *endpoint) Recover(ctx context.Context, req ctrl.Request, chaos v1alpha1
 		r.Log.Error(err, "fail to get service")
 		return err
 	}
-	r.Log.Info("Cancel DNS chaos to DNS service", "ip", service.Spec.ClusterIP)
+	err := r.Log.Info("Cancel DNS chaos to DNS service", "ip", service.Spec.ClusterIP)
+	if err != nil {
+		return err
+	}
```

Comment:
`r.Log.Info` does not return any error, it might be something wrong with the static code analyzer. 😅